README.md

```diff
@@ -152,12 +163,12 @@ If you like this project please give it a star
 - Gregoire Vella (Former product designer).
```

Comment:
- [ ] Please make sure to give proper credit to the author of the design as requested in the project requirements. You should also add a link to their profile on Behance.
  - You can replace the above text with this: `design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella)`